src/routes/widgets/handlers.js

```diff
@@ -22,7 +22,7 @@ export async function getWidget(req, res) {
     }
 
     // This returns the query object with "region" property filtered by user permissions
-    const query = await setReadRegions(req.query, req.session.userId, true);
+    const query = await setReadRegions(req.query, req.session.userId, false);
```

Comment:
I ran into an issue with this param looking at Adam's PR adding filters to the landing page. I think we can remove the param completely after both PRs make it up and just always assume `false`

Reply:
Actually I think we'll need to change it here, I only get a single region in the bottom table when I'm a central user. https://github.com/HHS/Head-Start-TTADP/blob/main/src/routes/activityReports/handlers.js#L511